common/src/point_types.cpp

```diff
@@ -375,6 +375,30 @@ namespace pcl
     return (os);
   }
 
+  std::ostream&
+  operator << (std::ostream& os, const GASDSignature512& p)
+  {
+    for (int i = 0; i < 512; ++i)
+    os << (i == 0 ? "(" : "") << p.histogram[i] << (i < 511 ? ", " : ")");
```

Comment:
Indentation here and in other operators below.

Reply:
I was not able to find the indentation problem here, would you please point this out to me?

Reply:
This is a loop body, therefore should be indented with 2 spaces with respect to the loop header.